game/core/src/main/java/com/gg/game/Model/Room.java

```diff
@@ -24,6 +24,17 @@ public Room(int width, int height) {
     private void generateEnemies() {
         enemies = new ArrayList<>();
         enemies.add(new Enemy(new Vector2()));
+        enemies.add(new Enemy(new Vector2(0f, 400f)));
+        enemies.add(new Enemy(new Vector2(400f, 0f)));
+        enemies.add(new Enemy(new Vector2(400f, 400f)));
```

Comment:
This will need to be modified after merging in the Vector/Point refactor
```suggestion
        enemies.add(new Enemy(new Point()));
        enemies.add(new Enemy(new Point(0f, 400f)));
        enemies.add(new Enemy(new Point(400f, 0f)));
        enemies.add(new Enemy(new Point(400f, 400f)));
```